yarn.lock

```diff
@@ -9731,7 +9738,7 @@ shellwords@^0.1.1:
   resolved "https://registry.yarnpkg.com/shellwords/-/shellwords-0.1.1.tgz#d6b9181c1a48d397324c84871efbcfc73fc0654b"
   integrity sha512-vFwSUfQvqybiICwZY5+DAWIPLKsWO31Q91JSKl3UYv+K5c2QRPzn0qzec6QPu1Qc9eHYItiP3NdJqNVqetYAww==
 
-signal-exit@3.0.2, signal-exit@TooTallNate/signal-exit#update/sighub-to-sigint-on-windows, signal-exit@^3.0.0, signal-exit@^3.0.2:
```

Comment:
@lucleray @TooTallNate We need to fix this. Perhaps we should publish a fork of signal-exit to avoid this issue?

Reply:
@styfle https://github.com/zeit/now/pull/3508

Reply:
Thanks for reporting 🙂